.github/workflows/tests.yml

```diff
@@ -15,7 +15,9 @@ jobs:
       - name: Select Xcode 11.2
         run: sudo xcode-select -s /Applications/Xcode_11.2.app
       - name: Build and Test
-        run: bazelisk test --local_test_jobs=1 //...
+        run: |
+          bazelisk test --local_test_jobs=1  //tests/macos/...
+          bazelisk test --local_test_jobs=1 --apple_platform_type=ios //tests/ios/...
```

Comment:
can we add something here to ensure that everything in the repo gets built, not just stuff in these two packages?

Reply:
I can not think of any way. Any ideas?

Reply:
I'd just throw in a `bazelisk build //...` in there before the tests, bazel will cache everything anyway.